cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlApp.java

```diff
@@ -0,0 +1,175 @@
+    String webDir = _config.getString(KafkaCruiseControlConfig.WEBSERVER_UI_DISKPATH);
+    String sessionPath = _config.getString(KafkaCruiseControlConfig.WEBSERVER_SESSION_PATH);
+    System.out.println(">> ********************************************* <<");
+    System.out.println(">> Application directory            : " + System.getProperty("user.dir"));
+    System.out.println(">> REST API available on            : " + webApiUrlPrefix);
+    System.out.println(">> Web UI available on              : " + uiUrlPrefix);
+    System.out.println(">> Web UI Directory                 : " + webDir);
+    System.out.println(">> Cookie prefix path               : " + sessionPath);
+    System.out.println(">> Kafka Cruise Control started on  : " + serverUrl());
+    System.out.println(">> CORS Enabled ?                   : " + corsEnabled);
+    System.out.println(">> ********************************************* <<");
+  }
+
+  private static KafkaCruiseControlConfig readConfig(String propertiesFile) throws IOException {
+    Properties props = new Properties();
+    try (InputStream propStream = new FileInputStream(propertiesFile)) {
+      props.load(propStream);
+    }
+
+    return new KafkaCruiseControlConfig(props);
+  }
+
+  private void setupWebUi(ServletContextHandler contextHandler) {
+    // Placeholder for any static content
+    String webuiDir = _config.getString(KafkaCruiseControlConfig.WEBSERVER_UI_DISKPATH);
+    String webuiPathPrefix = _config.getString(KafkaCruiseControlConfig.WEBSERVER_UI_URLPREFIX);
+    DefaultServlet defaultServlet = new DefaultServlet();
+    ServletHolder holderWebapp = new ServletHolder("default", defaultServlet);
+    // holderWebapp.setInitParameter("org.eclipse.jetty.servlet.Default.dirAllowed", "false");
+    holderWebapp.setInitParameter("resourceBase", webuiDir);
+    contextHandler.addServlet(holderWebapp, webuiPathPrefix);
+  }
+
+  private NCSARequestLog createRequestLog() {
+    boolean accessLogEnabled = _config.getBoolean(KafkaCruiseControlConfig.WEBSERVER_ACCESSLOG_ENABLED);
+    if (accessLogEnabled) {
+      String accessLogPath = _config.getString(KafkaCruiseControlConfig.WEBSERVER_ACCESSLOG_PATH);
+      int accessLogRetention = _config.getInt(KafkaCruiseControlConfig.WEBSERVER_ACCESSLOG_RETENTION_DAYS);
+      NCSARequestLog requestLog = new NCSARequestLog(accessLogPath);
+      requestLog.setRetainDays(accessLogRetention);
+      requestLog.setLogLatency(true);
+      requestLog.setAppend(true);
+      requestLog.setExtended(false);
+      requestLog.setPreferProxiedForAddress(true);
+      return requestLog;
+    } else {
+      return null;
+    }
+  }
+
+  private ServletContextHandler createContextHandler() {
+    // Define context for servlet
+    String sessionPath = _config.getString(KafkaCruiseControlConfig.WEBSERVER_SESSION_PATH);
+    ServletContextHandler context = new ServletContextHandler(ServletContextHandler.SESSIONS | ServletContextHandler.SECURITY);
```

Comment:
On a second thought, I think we don't need to use that option. It only affects `getSecurityHandler`: if the server is not started yet and there is no security handler set, it will create a default one if this option is set. Otherwise it'll just return what's been set.
In our case if there are no security configured, the `getSecurityHandler` shouldn't create one but if there is a `SecurityProvider` configured, we'll set one anyway.